addons/uis.md

```diff
@@ -21,6 +23,13 @@ title: User Interfaces
 | [iOS App]({{docu}}/addons/uis/apps/ios.html) | The native iOS app to access openHAB on the go. |
 | [Windows 10 App]({{docu}}/addons/uis/apps/windows.html) | The native Windows 10 app to access openHAB on the go. |
 
+## Iconsets
+
+Basic UI, Classic UI and all apps present a graphical user interface as defined in [Sitemaps]({{base}}/configuration/sitemaps.html) by the help of [Items]({{base}}/configuration/items.html).
```

Comment:
Consider, "Basic UI, Classic UI and all **openHAB** apps present a graphical user interface as defined in Sitemaps**, with ** the help of Items.